samcli/commands/build/command.py

```diff
@@ -0,0 +1,151 @@
+"""
+CLI command for "build" command
+"""
+
+import os
+import logging
+import click
+
+from samcli.commands.exceptions import UserException
+from samcli.yamlhelper import yaml_dump
+from samcli.cli.main import pass_context, common_options as cli_framework_options, aws_creds_options
+from samcli.commands._utils.options import template_option_without_build, docker_common_options, \
+    parameter_override_option
+from samcli.commands.build.build_context import BuildContext
+from samcli.lib.build.app_builder import ApplicationBuilder, UnsupportedRuntimeException, \
+    BuildError, UnsupportedBuilderLibraryVersionError
+
+LOG = logging.getLogger(__name__)
+
+
+HELP_TEXT = """
+Use this command to build your AWS Lambda Functions source code to generate artifacts that can run on AWS Lambda.\n
+\b
+Supported Resource Types
+------------------------
+1. AWS::Serverless::Function\n
+2.AWS::Lambda::Function\n
+\b
+Supported Runtimes
+------------------
+1. Python2.7\n
+2. Python3.6\n
+\b
+Usage
+-----
+In your SAM template, specify the path to your function's source code
+in the resource's Code or CodeUri property.
+\b
+To build on your workstation, run this command in folder containing
+SAM template. Built artifacts will be written to .aws-sam/build folder
+$ sam build\n
+\b
+To build inside a AWS Lambda like Docker container
+$ sam build --use-container
+\b
+To build & run your functions locally
+$ sam build && sam local invoke
```

Comment:
I love how simple this is going to make things. 🎆 🍾

Reply:
I know! I am super excited :)